[V3] Migrating to AsyncLocalStorage. (#13)
odedglas/node-execution-context: CHANGELOG.md

# CHANGELOG

## 3.0.0 (January 03, 2021)

> - Introduce `AsyncLocaleStorage` implementation based for node `12.7.0` and above.
> - `AsyncHooksContext` domains now created by default under they hook and do not requires consumer to defined it's name.
> - `update` API changed to `set`.
> - `create` API stop treads `context` as objects by design.

Reviewer: ```suggestion
- Introduces `AsyncLocaleStorage` based implementation for node `12.7.0` and above.
- `AsyncHooksContext` domains are now created by default under the hood and no longer require consumers to supply a name.
- `update` API changed to `set`.
- `create` API no longer treats`context` as JS objects.
```
